src/utils/time.ts

```diff
@@ -0,0 +1,17 @@
+export const timeAgo = (timestamp: number): string => {
```

Comment:
Same as before, remember that functions should be verbs: `const getTimeFromNow = ...`